Support attribute condition (#34)
newrelic-experimental/entity-synthesis-definitions: README.md

  - 1 to 36 standard ascii characters, excluding control chars (codes: 32-126).
  - If you suspect that your identifiers may not fulfil our length requirements, set the optional `encodeIdentifierInGUID` field to true. 
- The definition needs to provide enough information to differentiate this entity

>  from others. It cannot be a subset nor a superset of any existing definition. 

Reviewer: ```suggestion
  from others. It cannot be a subset nor a superset of any existing definition. 
```

> - If the names of your telemetry attributes are too generic you can define a condition on an attribute that needs to be met for the entity to be synthesized. 

Reviewer: Maybe "a condition to match on the value of an attribute" is clearer?

> - If the names of your telemetry attributes are too generic you can define a condition on an attribute that needs to be met for the entity to be synthesized. 

Reviewer: ```suggestion
- If the names of your telemetry attributes are too generic, you can define a condition to match on the value of an attribute. 
```
